desktop/apps/artwork/client/index.coffee

```diff
@@ -34,7 +34,13 @@ module.exports =
   setup: setup = (context = {}) ->
     if context.__typename is 'ArtworkContextAuction'
       query: """
-          query artwork($id: String!, $isClosed: Boolean!, $auctionId: ID) {
+          query artwork(
+            $id: String!,
+            $isClosed: Boolean!,
+            $auctionId: ID,
+            $saleSize: Int = 4,
+            $saleSort: SaleSorts = CREATED_AT_ASC
```

Comment:
I would use one of the `TIMELY_AT` sorts: https://github.com/artsy/metaphysics/blob/3bfc10db1aa2c8813d91bff8e78d398a7cfb9a65/schema/sale/sorts.js#L43

This will sort based on the sale's `live_start_at` or `end_at`, so the most _relevant_ (or 'timely') auctions at a given time. `created_at` is sort of arbitrary based on when the ops team created the auction, and not necessarily tied to when it's most important.